src/scales/scale.logarithmic.js

```diff
@@ -198,29 +199,23 @@ module.exports = function(Chart) {
 			} else {
 				// Bottom - top since pixels increase downward on a screen
 				innerDimension = me.height;
-				if (start === 0 && !tickOpts.reverse) {
-					range = helpers.log10(me.end) - helpers.log10(me.minNotZero);
-					if (newVal === start) {
-						pixel = me.bottom;
-					} else if (newVal === me.minNotZero) {
-						pixel = me.bottom - innerDimension * 0.02;
+				if (start === 0 || end === 0) {
+					var zero = tickOpts.reverse ? me.top : me.bottom;
+					var min = Math.pow(10, Math.floor(helpers.log10(me.minNotZero)));
+					var max = tickOpts.reverse ? start : end;
+					range = helpers.log10(max) - Math.floor(helpers.log10(me.minNotZero));
+					var diff = innerDimension * 0.04 + (innerDimension * 0.96/ range * (helpers.log10(newVal) - Math.floor(helpers.log10(me.minNotZero))));
+					if (newVal === 0) {
+						pixel = zero;
+					} else if (newVal === min) {
+						pixel = tickOpts.reverse ? zero + innerDimension * 0.04 : zero - innerDimension * 0.04;
```

Comment:
My concern here is that the heuristic of 4% (` * 0.04`) isn't guaranteed to work for all charts, particularly small ones. What may need to happen instead is that `innerDimension` is reduced by `fontSize + spacing` to always make room for the 0 tick. 

The axis would need to store where exactly 0 is so that when `getPixelForValue(0)` is run, we return the right thing.

Thoughts?